MATCHERS.md

```diff
@@ -70,6 +71,36 @@ state: s.enum({values: ['NSW', 'VIC'], name:'state'})
 state: s.enum({values: ['NSW', 'VIC'], name:'state', verbose: true})
 ```
 
+## exactObject
+
+Match an exact object.
+This matcher will not error if any optional properties are left out.
+This matcher will error if any extra properties passed in.
+
+Each property must have a corresponding matcher, and they will be called recursively.
+
+```js
+// match exact object
+person: 'exactObject'
+person: s.exactObject()
```

Comment:
Good point, too much copy and paste :)

I originally wanted it as an extra argument to the object matcher, but when myself and @rprieto looked at the code, it didn't seem like something that could be done without having to do

```coffee
s.object(
  schema: 
    name: 'string'
    address: 'string'
  exactArguments: true
)
```

or changing the signature of matchers to have 2 arguments, which I'm not sure we want to do